src/features/user/MFV2/ContributionsMap/index.tsx

```diff
@@ -49,7 +52,11 @@ function ContributionsMap() {
       {...mapState}
       onMove={(e) => setViewState(e.viewState)}
       attributionControl={false}
-    ></Map>
+    >
+      <CustomMarkers />
```

Comment:
Personally, I don't like using "Custom" in the name. It's too generic and tends to get overused